package.json

```diff
@@ -34,10 +34,10 @@
     "mdcss": "^1.5.2",
     "mdcss-theme-github": "^2.4.2",
     "mocha": "^8.1.3",
-    "next-standard-version": "^2.1.3",
+    "next-standard-version": "^1.0.1",
     "postcss": "^8.1.0",
     "postcss-cli": "^8.0.0",
-    "sass": "^1.26.11",
+    "sass": "^1.35.1",
```

Comment:
I think this should be listed in `peerDependencies` so npm can let consumers know they need to meet this sass version to use the package. You could add it with `npm i --save-peer sass@1.33.0`.

I'm assuming v1.33 because I think that's when it was introduced - https://sass-lang.com/documentation/breaking-changes/slash-div#transition-period. But I don't know what other features of sass we may be using that could affect what peer version the package should specify.

Reply:
TIL about Peer Dependencies. Thanks, @iAmNathanJ! We were only getting errors for the `math.div` change, so I think that will be good. Thanks!

Reply:
@iAmNathanJ I did some research into `peerDependencies`. I assume that having it listed as a peer will install when running from the main project, and only alert users about the minimum requirement situation. And that it is not necessary to have `sass` listed in both `peerDependencies` and `devDependencies`.

Reply:
@zastrow Thank you for adding this! Curious, what’s your `npm` version? I think peerDependencies are only installed automatically for npm v7. When using v6 I don’t get sass installed so all the scripts that use `sass` are failing. I have verified the install works for `npm@7`. Since `engines` has "npm": ">=6.0.0" then you might need this back in the devDeps.